MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -162,6 +168,9 @@ public bool NeedsRebuild(PipelineManager manager, PipelineBuildEvent cachedEvent
             if (cachedEvent == null)
                 return true;
 
+            // jcf: Not sure if we need to deal with 'file exists in output but with different case'
+            //      here, or if that should be done elsewhere.
+
```

Comment:
Yes I think we should.  This should handle all cases of a file being dirty.

Now you could argue a different case is actually a different file... which is how things work on non-Windows OSs.  However for ease of use and portability, I really think MG and the pipeline should try to work as file case does not matter.